Promact.Oauth.Server/src/Promact.Oauth.Server/Consent/ConsentOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Promact.Oauth.Server.Consent
+{
+    public class ConsentOptions
```

Comment:
Should be in Utility folder and We don't use statics. So, should have an Interface and have no static variables